Dockerfile

```diff
@@ -7,22 +7,32 @@ ENV PYTHONUNBUFFERED 1
 WORKDIR /app
```

Comment:
Not related to your changes directly but when using helsinkitest images, WORKDIR /app is already set for you.

Reply:
I think it's better to be explicit here.